README.md

```diff
@@ -82,6 +82,54 @@ You may optionally include an OpenAiOrganization if you have multiple Organizati
 var api = new OpenAiApi(new ApiAuthentication("YOUR_API_KEY", "org-yourOrgHere"));
 ```
 
+### Audio Transcription
+
+You can transcribe an audio file with OpenAiNg using the following code snippet:
```

Comment:
great, thanks. This is also somewhat described here: https://github.com/lofcz/OpenAiNg/blob/master/OpenAiNg.Demo/SpeechDemo.cs